/home/liu/miniconda3/envs/GAIC/lib/python3.10/site-packages/open_clip/factory.py:388: UserWarning: These pretrained weights were trained with QuickGELU activation but the model config does not have that enabled. Consider using a model config with a "-quickgelu" suffix or enable with a flag.
  warnings.warn(
Traceback (most recent call last):
  File "/home/liu/miniconda3/envs/GAIC/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/liu/miniconda3/envs/GAIC/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/raid/dannyliu/GAI_Detection/GenAI_Detection/src/evaluation/eval_cvfid_benchmark.py", line 193, in <module>
    main(args)
  File "/raid/dannyliu/GAI_Detection/GenAI_Detection/src/evaluation/eval_cvfid_benchmark.py", line 118, in main
    ckpt = torch.load(args.ckpt, map_location="cpu")
  File "/home/liu/miniconda3/envs/GAIC/lib/python3.10/site-packages/torch/serialization.py", line 1479, in load
    with _open_file_like(f, "rb") as opened_file:
  File "/home/liu/miniconda3/envs/GAIC/lib/python3.10/site-packages/torch/serialization.py", line 759, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/liu/miniconda3/envs/GAIC/lib/python3.10/site-packages/torch/serialization.py", line 740, in __init__
    super().__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'results/stage2/best_stage2.pt'
